src/components/Dropdown/Dropdown.tsx

```diff
@@ -162,20 +172,33 @@ function useFocusContextInstance(
   lastTypedOrSubmittedValue: string,
   setValue: (newValue: string) => void,
   screenReaderKey: number,
-  setScreenReaderKey: (newKey: number) => void
+  setScreenReaderKey: (newKey: number) => void,
+  alwaysSelectOption: boolean
 ): FocusContextType {
   const [focusedIndex, setFocusedIndex] = useState(-1);
   const [focusedValue, setFocusedValue] = useState<string | null>(null);
   const [focusedItemData, setFocusedItemData] = useState<Record<string, unknown> | undefined>(undefined);
 
+  useEffect(() => {
+    if (alwaysSelectOption && items.length > 0) {
+      setFocusedIndex(0);
+      setFocusedItemData(items[0].itemData);
```

Comment:
if someone types something in the filter search input, we set the first option as active/focused. but it seems like that would cause the filter to be applied if the user clicks out of the dropdown without explicitly selecting or focusing on the option, but then clicks the Apply Filters button. is that the desired behavior?

Reply:
The very last scenario I listed here still seems a little odd. If the user types something, navigates through the dropdown, but then changes the input text before clicking out, would we want a filter to be selected? since the last time they changed the input, they haven't interacted with the dropdown. so, maybe the desired behavior would be to not select the filter unless the user interacts with the dropdown after the last time they type? this might be a question for product